dataland-frontend/src/components/pages/ApiKeysPage.vue

```diff
@@ -76,11 +76,7 @@
             </template>
           </MessageComponent>
 
```

Comment:
ok

Reply:
Looks good 👍 

I just noticed that we still use our class ".d-center-div" a lot in different components, mainly to put the loading circle icon in the middle of the page.
I think we could use your MiddleCenterDivWrappe.vue in all of these cases, and then drop the ".d-center-div" CSS-class completely from our code.

This would be more consistent.

What do you think?